docs/specification/draft/client/elicitation.mdx

```diff
@@ -8,15 +8,19 @@ title: Elicitation
 
 <Note>
 
-Elicitation is newly introduced in this version of the MCP specification and its design may evolve in future protocol versions.
+The design of the Elicitation capability may evolve in future protocol versions.
 
 </Note>
 
 The Model Context Protocol (MCP) provides a standardized way for servers to request additional
 information from users through the client during interactions. This flow allows clients to
 maintain control over user interactions and data sharing while enabling servers to gather
 necessary information dynamically.
-Servers request structured data from users with JSON schemas to validate responses.
+
+Elicitation supports two modes:
+
+- **Form mode** (in-band): Servers can request structured data from users with optional JSON schemas to validate responses
```

Comment:
For simplification purposes, I am not sure it's worth calling them "in-band" and "out-of-band" - the modes themselves should be sufficient.